### PR TITLE
Escape special symbols

### PR DESCRIPTION
**Pull request by misteraverin**

### CODE REVIEW

File: translation/models/translation_services.py [hunk 1]
```diff
@@ -8,6 +8,24 @@
...
+def escape_symbols(texts):
+    for idx, step_text in enumerate(texts):
+        step_text = step_text.replace(";", "%3B")
+        step_text = step_text.replace("&", "%26")
+        step_text = step_text.replace("+", "%2B")
+        texts[idx] = step_text
+
+
+# :param array of texts
+# delete accidently happened commas. E.g. "Blue, Sky, Orage" -> "Blue SKy, Orange" -> ["Blue", "Sky,", "Orange"]
```
Comment (KirillMakarov) [inline translation/models/translation_services.py]: Please, fix typos in description of the method

File: translation/models/translation_services.py [hunk 2]
```diff
@@ -8,6 +8,24 @@
...
+def delete_strange_semicolons(texts):
```
Comment (KirillMakarov) [inline translation/models/translation_services.py]: semicolons is `;`, but method deletes commas.

File: translation/models/translation_services.py (hunk 1 above)
Comment (KirillMakarov) [inline translation/models/translation_services.py]: What is precondition for texts? is it string or array of strings?

File: translation/models/translation_services.py [hunk 3]
```diff
@@ -8,6 +8,24 @@
 from .translation import TranslatedStep, TranslatedLesson, TranslatedCourse, StepSource, TranslatedStepSource
 
 
+# :param: array of texts
+#  make array with escaped symbols (";", "&", "+")
+def escape_symbols(texts):
```
Comment (KirillMakarov) [inline translation/models/translation_services.py]: why just this symobols are escaped?  May be https://docs.python.org/3.4/library/re.html#re.escape is more suitable?

File: translation/models/translation_services.py (hunk 2 above)
Comment (KirillMakarov) [inline translation/models/translation_services.py]: Why comma after sky is not removed here?

**Review by KirillMakarov — CHANGES_REQUESTED** [review]

File: translation/models/translation_services.py (hunk 3 above)
Comment (misteraverin) [reply translation/models/translation_services.py]: @KirillMakarov actually we don't want to escape, we want ecode and `re.escape` act like this `& -> \\ &` which is useless

**Pull request merged**